pulpcore/app/management/commands/runserver-coverage.py

```diff
@@ -0,0 +1,17 @@
+import atexit
```

Comment:
Putting this here will cause it to also be on production systems which doesn't seem right to me. That being said all test environments would benefit from it so reusability is good in those environments.

Does moving this into a Travis script make sense? Is my concern a real one?

Reply:
Using `coverage run pulp-manager runserver` did not work, i think. That is why i searched for a way to insert it into the command. If we can manipulate the corresponding search path, we can place it elsewhere. I do not know how.